--- origsrc/wordplay-0.3/wordplay.c	2007-08-14 02:48:14.000000000 -0500
+++ src/wordplay-0.3/wordplay.c	2007-08-18 22:26:23.437500000 -0500
@@ -2,7 +2,9 @@
 #include <stdlib.h>
 #include <string.h>
 #include <ctype.h>
+#ifndef __CYGWIN__
 #include <values.h>
+#endif
 #include <stdarg.h>
 #include <errno.h>
 #include <unistd.h>
@@ -553,7 +555,7 @@
     /* Set the age of this puzzle back to 0, and age all other puzzles */
     p->age = 0;
     for (i=0; i < global.num_puzzles; i++) {
-        if (i != global.cur_index && global.puzzles[i].age < MAXSHORT)
+        if (i != global.cur_index && global.puzzles[i].age < G_MAXSHORT)
             global.puzzles[i].age++;
     }
 
